docs/deployment/FPC_on_Azure.md

```diff
@@ -0,0 +1,139 @@
+
+- [How to use an Azure Confidential Computing Instance to run FPC in HW mode](#how-to-use-an-azure-confidential-computing-instance-to-run-fpc-in-hw-mode)
+  - [Creating the Confidential Computing Instance on Azure.](#creating-the-confidential-computing-instance-on-azure)
+  - [Prerequisites](#prerequisites)
+  - [Registering for the SGX Attestation Service Utilizing EPID](#registering-for-the-sgx-attestation-service-utilizing-epid)
+    - [Known Issues](#known-issues)
+  - [Building the FPC dev container](#building-the-fpc-dev-container)
+  - [Final preparation of the dev-container](#final-preparation-of-the-dev-container)
+
+## Creating the Confidential Computing Instance on Azure.
+Use the [Quick Create Portal](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal) to create
+the virtual machine.
+Use the following parameters:
+* OS = *Linux (ubuntu 20.04)*
+* Size = Standard DC2s v2 (2 vcpus, 8 GiB memory)
+
+## Prerequisites
+Connect to your instance using ssh and execute the following commands
+```bash
+#update machine
+sudo -i
+apt-get update
+apt-get upgrade
+apt install -y docker docker-compose golang make
+exit
+#install sgx environment
+#1.add sgx repo + key and then install packages form this repository
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+sudo apt-get update
+sudo apt -y install libssl-dev libsgx-enclave-common libsgx-enclave-common-dev libsgx-ae-qe3 libsgx-ae-qve libsgx-epid libsgx-launch libsgx-pce-logic libsgx-qe3-logic libsgx-quote-ex libsgx-uae-service libsgx-urts
+sudo reboot
+```
+After reboot is completed connect to your vm again.
+```bash
+#check if asemd service is running
+sudo systemctl status aesmd.service
+```
+```bash
+sudo usermod -aG docker $(whoami)
+go env -w GO111MODULE=auto
+
+echo 'export GOPATH=$HOME/go' >> .bashrc
+echo 'export GOBIN=$GOPATH/bin' >> .bashrc
+echo 'export PATH=$PATH:$GOBIN' >> .bashrc
+
+sudo reboot
+``` 
+After reboot is completed connect to your vm again.
+```bash
+mkdir -p $GOPATH
+cd $GOPATH
+go get github.com/hyperledger/fabric-private-chaincode
+cd  ~
+echo 'export FPC_PATH=$GOPATH/src/github.com/hyperledger/fabric-private-chaincode' >> .bashrc
+#you can also re-log than you don't need to set it
+export FPC_PATH=$GOPATH/src/github.com/hyperledger/fabric-private-chaincode 
+```
+
+## Registering for the SGX Attestation Service Utilizing EPID
+* Get a account [here](https://www.intel.com/content/www/us/en/forms/basic-intel-registration.html)
+* Sign in.
+* Once you are signed in go [here](here: https://api.portal.trustedservices.intel.com/EPID-attestation)
+* Subscribe for development unlikable.
```

Comment:
I guess, alternatively, one could also use linkable attestation if wanted?!

Reply:
Yeah that is true i guess. I have only tried with unlinkable.
Also the later steps create the file spid_type.txt with "epid-unlinkable" so thats why I mention it here. 
I will add a sentence, that both has to be changed in order to use linkable attestation.